cmd/agent/agent.go

```diff
@@ -69,7 +70,7 @@ func loop(c *cli.Context) error {
 		)
 	}
 
-	zerolog.SetGlobalLevel(zerolog.WarnLevel)
+	zerolog.SetGlobalLevel(zerolog.InfoLevel)
```

Comment:
Ideally this should be settable by env variable

Reply:
You can set it. This is the value it uses in case you haven't specified it.